src/scenario_tests.rs

```diff
@@ -249,36 +213,37 @@ mod dst_tests {
     #[test]
     fn rpc_expect_candidate() {
         run_test(
-            "Get RPC ExpectCandidate",
+            "",
```

Comment:
Blank description?

Reply:
Yes, there is really not much to add that is not already in the test name.